django/http/response.py

```diff
@@ -299,10 +299,15 @@ def __init__(self, content=b'', *args, **kwargs):
         self.content = content
 
     def __repr__(self):
-        return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+        if 'Content-Type' in self:
+            return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+                'cls': self.__class__.__name__,
+                'status_code': self.status_code,
+                'content_type': self['Content-Type'],
+            }
+        return '<%(cls)s status_code=%(status_code)d>' % {
```

Comment:
This `content_type_bit` would be the same for three response object. Do you think it would be appropriate to add `_content_type_for_repr` property to `HttpResponseBase` for that?

I prefer `str.format` as well, but I thought that keeping with existing code style is better in this case.